src/App.jsx

```diff
@@ -33,8 +40,14 @@ function App() {
       }
       
       const data = await response.json();
+
+      // Stores the number of completed todos (first cell) and its id, and excludes it from fetching
+      setCompletedTodosCount(Number(data.records[0].fields.Title));
+      setCompletedTodosCountId(data.records[0].id);
```

Comment:
It's not clear that these two things are working as intended, although you might have your airtable database set up differently.

For me, this does not trigger the reset button to display when getting over 15 todos. I'd think it'd look like this:
```js
setCompletedTodosCount(Number(data.records));
```

and without the:
```js
setCompletedTodosCountId(data.records[0].id);
```
entirely maybe? I'm not entirely sure how this all plays into the pet stuff, though. It's just a little hard to follow.